api/handler/triggers.go

```diff
@@ -51,13 +51,23 @@ func getAllTriggers(writer http.ResponseWriter, request *http.Request) {
 	}
 }
 
+// createTrigger handler creates moira.Trigger.
 func createTrigger(writer http.ResponseWriter, request *http.Request) {
 	trigger, err := getTriggerFromRequest(request)
 	if err != nil {
 		render.Render(writer, request, err) //nolint
 		return
 	}
 
+	var problems []dto.TreeOfProblems
+	if isNeedValidate(request) {
+		problems = validateTargets(request, trigger)
```

Comment:
why dont use := ? :)

Reply:
Can't use `:=` on 64th line, because need to write problems to reponse on 88th line. Can't use `:=` on 62th line, because need to validate only if `needValidate`